ee/get-support.md

```diff
@@ -45,6 +45,33 @@ This support dump only contains logs for the node where you're running the
 command. If your UCP is highly available, you should collect support dumps
 from all of the manager nodes.
 
+## Use the CLI to get a support dump for EE Basic
+
+To get the support dump from the EE Basic Engine CLI, use SSH to log into a node and run:
+
+```
+docker run --rm  \
+    -v /boot:/boot \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v /var/run/docker.pid:/var/run/docker.pid:ro \
+    -v /var/run/docker:/var/run/docker \
+    -v /var/lib/docker:/var/lib/docker \
+    -v /var/run/docker/netns:/var/run/docker/netns \
+    -v /var/lib/docker:/var/lib/docker \
```

Comment:
`/var/lib/docker` is provided twice

Reply:
Thanks! Just removed the duplicate line.